OpenRA.Mods.Common/Activities/Move/Move.cs

```diff
@@ -190,7 +190,7 @@ public override Activity Tick(Actor self)
 			if (IsCanceled && self.Location.Layer != CustomMovementLayerType.Tunnel)
 				return NextActivity;
 
-			if (mobile.IsTraitDisabled)
+			if (mobile.IsTraitDisabled || mobile.IsTraitPaused)
```

Comment:
Does line 376 below need this too?

Reply:
I'm not sure. It depends on what the desired behaviour is. Should for instance EMPed units freeze immediately when in between cells or should the unit always 'coast' to the next cell? I think you can argue either way.